### ISSUE TITLE
Video not available in your country.

### ISSUE THREAD

**Issue by jsveto** [Sandmann79/xbmc#184]
Hi, i installed amazon prime video (de) plugin and it works ok. but 95%+ videos will not play. Kodi reports error: Video not available in your country. Same videos play OK from my PC web interface.

Furthermore Kodi shows all videos from amazon including those which are not available in my country. For example Vikings are shown on Kodi but not on PC. Is there a way to only show videos available in my country?

**Comment by Sandmann79** [comment]
If you're not using Amazon VOD, you have to wait till the database update has finished. 
You can also try to rebuild the database by opening the  context menu at the tvshows menu entry and choose full television refresh.

BTW: Vikings is included in prime membership in Germany

**Comment by jsveto** [comment]
Hi, i did full television refresh and full DB rebuild as well. It took about 1h. However same videos still report not available in your country error.

For example i can watch Startup S01E01 from my web browser, but KODI reports error "not available in your country"

For example Man in the high castle plays OK both in web browser and KODI.

I have amazon.de prime membership (I am not from Germany, but other EU country).

**Comment by Sandmann79** [comment]
When you logged in at Addon you're used an VPN or changed the region to germany?

**Comment by jsveto** [comment]
no

**Comment by jsveto** [comment]
Amazon.de addon does not have region setup. I tried removing credentials, setting kodi timezone to germany and log in again. Anything i tried to do with Amazon.de addon didn't work. I still get "not available in your country" messages on most of the videos.

I tried amazon VOD plugin. I was able to log in. First time i opened plugin i saw some messages about shows/movies being found. However when i browse through television/movies menu, each submenu is empty (no shows). BUT! If i add shows to watchlist (from my PC) i can see them on Kodi in watchlist menu and able to play them. Is there some log i can check to see what is going on?

**Comment by Varstahl** [comment]
@Drahas Amazon VOD in France should work with Auto detection, you shouldn't manually set the region. Also you should be able to see the content in the language you set in PrimeVideo.com, so if you see stuff in german either you made a mistake in the settings or you're using VPNs or proxies.

**Comment by Drahas** [comment]
@Varstahl I set the region to Auto and I relog on Amazon and now I can't see any tv show or movies in the list (there is on error when I try to load series).
I didn't change other settings in Amazon VOD addon.
Should I set a VPN and which country for the VPN ?

**Comment by Varstahl** [comment]
@Drahas take the Kodi.log, poste it on pastebin, and open a new issue linking to it, so that I can take a look. You shouldn't need any kind of VPN, if there's a problem I need to check it out.

**Comment by Drahas** [comment]
Ok I will do it, Thx

**Comment by Sandmann79** [comment]
@jsveto is this issue still present?

**Comment by antohind** [comment]
Alright, 

Just after a little bit of help with the Amazon VOD addon I am using the latest version of OSMC on a Raspberry Pi 3 but constantly getting an error that media is unable to play in my country 

I have configured A vpn using OPenvpn set it to USA, Germany, UK and this will still not play can you please advise 

Regards

**Comment by Varstahl** [comment]
You can't really use a VPN afaik, you're limited not only by geofencing but also from account settings. If you have a Prime subscription you should access the appropriate website for your country.

**Comment by antohind** [comment]
OK so remove the VPN and set the region back to UK and your saying that should work ??

Thanks for the quick response

**Comment by Varstahl** [comment]
Yes, with Amazon VOD it should work straight out of the box. You might need to wait for the catalog update though, I'm told that it can take a few hours even on fast connections.

**Comment by antohind** [comment]
Cool will test again tomorrow 

Thanks very much
